packages/eslint-config-react-app/index.js

```diff
@@ -22,13 +22,39 @@
 // We blacklist the globals that we deem potentially confusing.
 // To use them, explicitly reference them, e.g. `window.name` or `window.status`.
 const restrictedGlobals = require('confusing-browser-globals');
+const readPkgUp = require('read-pkg-up');
+
+let hasJestDom = false;
+let hasTestingLibrary = false;
+
+try {
+  const pkg = readPkgUp.sync({ normalize: true });
+  const allDeps = Object.keys({
+    ...pkg.peerDependencies,
+    ...pkg.devDependencies,
+    ...pkg.dependencies,
+  });
+
+  hasJestDom = allDeps.includes('@testing-library/jest-dom');
+  hasTestingLibrary = allDeps.includes('@testing-library/react');
+} catch (error) {
+  // ignore error
+}
 
 module.exports = {
   root: true,
 
   parser: 'babel-eslint',
 
-  plugins: ['import', 'flowtype', 'jsx-a11y', 'react', 'react-hooks'],
+  plugins: [
+    'import',
+    'flowtype',
+    'jsx-a11y',
+    'react',
+    'react-hooks',
+    hasJestDom ? 'jest-dom' : null,
```

Comment:
```suggestion
    hasJestDom && 'jest-dom',
```

Reply:
Any reason why this is recommended? 🤔

Reply:
less code. accomplishes the same thing

Reply:
Using a ternary is how it's done in `kcd-scripts` and is how it's recommended by @kentcdodds itself, so going to leave it like this